proprietary/tokens/README.md

```diff
@@ -1,3 +1,141 @@
 <!-- @license CC0-1.0 -->
 
-# TODO: write readme
+# Amsterdam Design System: Tokens
+
+This package provides the design tokens of the Amsterdam Design System.
+Every token is a variable that stores a single visual design choice.
+Tokens exist for colours, typographic attributes, spacing lengths, border widths, animation properties, etc.
+Every token has a value that is valid in CSS.
+
+Some examples:
+
+```css
+:root {
+  --ams-color-blue: #004699;
+  --ams-text-font-weight-bold: 800;
+  --ams-border-width-medium: 0.125rem;
+  --ams-button-padding-inline: 1.5rem;
+  --ams-space-xl: clamp(2.25rem, 2.0625rem + 0.9375vw, 3rem);
+  --ams-text-input-invalid-border-color: var(--ams-color-red);
+}
+```
+
+All tokens together form a theme that encodes the entire branding of the City of Amsterdam.
+
+We use these tokens in our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react).
+Any web-based application for Amsterdam can implement the design system through this package.
+
+## Install the package
+
+Use this package alone if you want or need to use the design tokens directly in your project.
+Note that our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) provide more functionality and apply the tokens automatically.
+
+```sh
+pnpm add @amsterdam/design-system-tokens
+```
+
+## Import the tokens
+
+Find the tokens in the `dist` directory and import them into your project.
+
+Tokens are typically applied as custom properties in CSS.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+```
+
+### Root or class selector
+
+The stylesheets wrap the tokens into a `:root` selector.
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+### Compact mode
+
+Applications need less whitespace and smaller text sizes than websites.
+Add the compact stylesheet to override the values of these tokens.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+Dark mode will work the same way when it’s ready.
+
+### Other formats
+
+The tokens can be imported as Sass variables as well.
+
+```sass
+@import "@amsterdam/design-system-tokens/dist/index.scss"
+```
+
+To use the tokens in JavaScript, import the JSON file.
+
+```ts
+import tokens from "@amsterdam/design-system-tokens/dist/index.json";
+```
+
+## Use the tokens
```

Comment:
This section is CSS specific again, no? Maybe we should specify that.

```suggestion
## Use the CSS tokens
```

Reply:
Thanks, this made me restructure the entire piece 😎

All concepts (three layers, compact mode etc.) are now explained under ‘CSS’, followed by small sections on secondary usage in Sass and JavaScript.

The latter made me wonder whether we should still export the `.mjs` format – JSON seems easier to use.

Reply:
I think Robbert was working on a generic list of export types of tokens for the entire NLDS community, so I wouldn't mess too much with it now.